README.md

```diff
@@ -53,6 +53,9 @@ If you find a problem or missing feature, open an issue or a pull requests on ht
 
 To have a higher chance of issue being solved, please attach a log file. To record one, go to `settings wheel -> System settings -> Logging -> Enable Debug Logging` and follow the procedure at http://kodi.wiki/view/Log_file/Easy
 
-## Note for Raspberry Pi users:
+# Note for ARM users #
+## Nvidia Shield / most Android boxes ##
+- _Ambilight mode_ doesn't properly work with some HD codecs (usually x265 >1080p) when "allow hardware acceleration - Mediacodec (Surface)" is enabled.
+## Raspberry Pi's: ##
  - Save the add-on configuration by exiting Kodi before shutting down the Pi completely
- - Ambilight mode doesn't work on a Raspberry Pi due to the way it renders video
+ - _Ambilight mode_ doesn't work on a _Raspberry Pi_ due to the way it renders video
```

Comment:
Hi there, just found this project but I have a RPi3. Can you maybe add some more information here why exactly this is not working (Hardware acceleration?) and if it's true for all versions of the Raspberry Pi. Otherwise I might waste some time trying anyways :wink: 
And - at least for me - it would be interesting to know about alternative ARM devices which do not have this issue.